### PR TITLE
feat: clickhouselogsexporter updated to support v0.63

### PR DESCRIPTION
**Pull request by nityanandagohain**
Fixes https://github.com/SigNoz/engineering-pod/issues/816

### CODE REVIEW

File: exporter/clickhouselogsexporter/exporter.go [hunk 1]
```diff
@@ -144,10 +144,10 @@ func attributesToSlice(attributes pcommon.Map, forceStringValues bool) (response
 			switch v.Type().String() {
 			case "INT":
 				response.IntKeys = append(response.IntKeys, formatKey(k))
-				response.IntValues = append(response.IntValues, v.IntVal())
+				response.IntValues = append(response.IntValues, v.Int())
 			case "DOUBLE":
```
Comment (srikanthccv) [inline exporter/clickhouselogsexporter/exporter.go:148]: These case values have changed. Please check once. Also, don't use the string comparison instead, use the type itself.

**Review by srikanthccv — CHANGES_REQUESTED** [review]

File: exporter/clickhouselogsexporter/exporter.go (hunk 1 above)
Comment (nityanandagohain) [reply exporter/clickhouselogsexporter/exporter.go]: Aah thanks, missed it during testing. Have updated it.

**Review by srikanthccv — APPROVED** [review]

File: exporter/clickhouselogsexporter/factory.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ func NewFactory() component.ExporterFactory {
 	return component.NewExporterFactory(
 		typeStr,
 		createDefaultConfig,
-		component.WithLogsExporterAndStabilityLevel(createLogsExporter, component.StabilityLevelBeta),
+		component.WithLogsExporter(createLogsExporter, component.StabilityLevelBeta),
```
Comment (makeavish) [inline exporter/clickhouselogsexporter/factory.go:41]: Any specific reason to set stability to beta?

**Review by makeavish — APPROVED** [review]

File: exporter/clickhouselogsexporter/factory.go (hunk 2 above)
Comment (nityanandagohain) [reply exporter/clickhouselogsexporter/factory.go:41]: Not specific, just waiting to see if we can find any more issues before we declare it stable.

**Comment by srikanthccv** [comment]
I am going to pull these changes and complete my work.

**Pull request merged**